mintapi/signIn.py

```diff
@@ -41,13 +41,6 @@ class MFAMethodNotAvailableError(RuntimeError):
 BUTTON_CSS_SELECTORS_LABEL = "button_css_selectors"
 
 MFA_METHODS = [
-    {
```

Comment:
This needs to stay.

Reply:
I can leave soft-token alone and fix the selector and also change the method name so it matches if you are more comfortable with that. But I think that would make it a duplicate of the Authenticator dict...

Reply:
@burkematthew what are your thoughts on this?